frontend/shared-components/external-link/external-link.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import styles from "./external-link.module.css"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faExternalLinkAlt } from "@fortawesome/free-solid-svg-icons"
+
+interface ExternalLinkProps {
+  linkPath: string
+  linkText: string
+}
+
+export default function ExternalLink({ linkPath, linkText }: ExternalLinkProps) {
+  const { externalLink } = styles
+  return (
+    <a aria-label={`${linkText} external`} className={externalLink} href={linkPath}>
+      {linkText}
+      <FontAwesomeIcon aria-hidden="true" icon={faExternalLinkAlt} size="xs" />
```

Comment:
nit: I think this icon implies the link opens in a new tab. Since this is a web app i do think it makes sense for external links to open in new tabs. Can update the link attributes with `target="_blank" rel="noopener noreferrer"`

Reply:
That was in my first submission, but apparently forcing new tabs are controversial. Unfortunately my git problems prevented that discussion from appearing here, but Opinions were Expressed, so I took it out. We probably need a floor debate on the issue.

Reply:
alright--my understanding is that stateless pages should not open in new tabs but it's ok for web apps with state to open links in new tabs to avoid losing a user's progress. 🤷

Reply:
that works